### ISSUE TITLE
Forward slashes don't work under Windows

### ISSUE THREAD

**Issue by wilhelmer** [apenwarr/mkdocs-exclude#2]
My MkDocs project runs on Windows, and forward slashes in path names don't work for me:
```
plugins:
  - exclude:
      glob:
        - "exclude/this/path/*"
```
This worked for me, but I guess it will only work on Windows:
```
plugins:
  - exclude:
      glob:
        - "exclude\\this\\path\\*"
```
Please fix this for cross-platform compatibility. We should be able to use forward slashes on all platforms, just like with other plugins.

**Comment by apenwarr** [comment]
thanks for the report.  Should be fixed in 1.0.2.

**Comment by wilhelmer** [comment]
Yep, it's fixed. Thanks!
